site_scons/daos_build.py

```diff
@@ -25,7 +25,10 @@ def add_rpaths(env, install_off, set_cgo_ld, is_bin):
     rpaths = env.subst("$RPATH_FULL").split()
     prefix = env.get("PREFIX")
     for rpath in rpaths:
-        if install_off is None:
+        if rpath.startswith('/usr'):
```

Comment:
(pylint-no-else-continue) Unnecessary "elif" after "continue"

Reply:
Is this still needed ?

Reply:
probably not, I will try to remove as part of https://jira.hpdd.intel.com/browse/DAOS-7960